runtime/parachain/src/lib.rs

```diff
@@ -916,8 +927,113 @@ impl ethereum_light_client::Config for Runtime {
 impl pallet_utility::Config for Runtime {
     type Event = Event;
     type Call = Call;
-    type WeightInfo = pallet_utility::weights::SubstrateWeight<Self>;
     type PalletsOrigin = OriginCaller;
+    type WeightInfo = pallet_utility::weights::SubstrateWeight<Self>;
+}
+
+pub const CONTRACTS_DEBUG_OUTPUT: bool = true;
+
+parameter_types! {
+    pub const ContractDeposit: u64 = 16;
+
+    // The lazy deletion runs inside on_initialize.
+    pub const DeletionQueueDepth: u32 = 1024;
+    pub const DeletionWeightLimit: Weight = 500_000_000_000;
+    pub Schedule: pallet_contracts::Schedule<Runtime> = {
+        let mut schedule = pallet_contracts::Schedule::<Runtime>::default();
+        // We decided to **temporarily* increase the default allowed contract size here
+        // (the default is `128 * 1024`).
+        //
+        // Our reasoning is that a number of people ran into `CodeTooLarge` when trying
+        // to deploy their contracts. We are currently introducing a number of optimizations
+        // into ink! which should bring the contract sizes lower. In the meantime we don't
+        // want to pose additional friction on developers.
+        schedule.limits.code_len = 256 * 1024;
+        schedule
+    };
+}
+
+impl pallet_contracts::Config for Runtime {
+    type Time = Timestamp;
+    type Randomness = RandomnessCollectiveFlip;
+    type Currency = Balances;
+    type Event = Event;
+    type Call = Call;
+    /// The safest default is to allow no calls at all.
+    ///
+    /// Runtimes should whitelist dispatchables that are allowed to be called from contracts
+    /// and make sure they are stable. Dispatchables exposed to contracts are not allowed to
+    /// change because that would break already deployed contracts. The `Call` structure itself
+    /// is not allowed to change the indices of existing pallets, too.
+    type CallFilter = frame_support::traits::Nothing;
+    type WeightPrice = pallet_transaction_payment::Pallet<Self>;
+    type WeightInfo = pallet_contracts::weights::SubstrateWeight<Self>;
+    type ChainExtension = ();
+    type Schedule = Schedule;
+    type ContractDeposit = ContractDeposit;
+    type CallStack = [pallet_contracts::Frame<Self>; 31];
+    type DeletionQueueDepth = DeletionQueueDepth;
+    type DeletionWeightLimit = DeletionWeightLimit;
+}
+
+// EVM
+parameter_types! {
+    pub const ChainId: u64 = 33;
```

Comment:
can we use a const for the value here?

Reply:
Yup, this is pretty much a constant which is set either directly in runtime (as [Acala](https://github.com/AcalaNetwork/Acala/blob/969878303c90f9a019df32b017e87c76c6b5d73e/runtime/acala/src/lib.rs#L1391) does for example) or set during the genesis build (like [moonbeam](https://github.com/PureStake/moonbeam/blob/7e84e15003cca1c8d9b72b8fd3e6132283f43993/node/service/src/chain_spec/moonbeam.rs#L143)). We could use either approach, but I think having it as a constant and setting it directly in the runtime config makes sense for now. Plus that number is more or less dummy until we become a parachain. 

One note would be that the `ChainId`s should be different for parachain and standalone, and I'll correct that on the following commit